prow/scripts/cluster-integration/helpers/get-letsencrypt-cert.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+ printf "\nChecking if certificate is already in GCP Bucket."
+      mkdir -p ./letsencrypt/live/"${DOMAIN}"
+ if [[ $(gsutil ls gs://kyma-prow-secrets/nightly-gke-tls-integration-app-client-cert.encrypted) ]];
+ then
+    printf "\nCertificate/privatekey exists in vault. Downloading..."
+
+    cp /etc/credentials/sa-gke-kyma-integration/service-account.json letsencrypt
```

Comment:
Why service account is copied to `letsencrypt` directory? It is not needed as you only download files from bucket.